test/ListUploadsTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Storj\Uplink\Test;
+
+use DateTimeImmutable;
+use PHPUnit\Framework\TestCase;
+use Storj\Uplink\ListObjectsOptions;
+use Storj\Uplink\ObjectInfo;
+use Storj\Uplink\UploadInfo;
+
+class ListUploadsTest extends TestCase
+{
+    public function testListUploads(): void
+    {
+        $project = Util::emptyProject();
+        $project->createBucket('phpunit');
+
+        $uploads = iterator_to_array($project->listUploads('phpunit'));
+        self::assertCount(0, $uploads);
+
+        $upload = $project->uploadObject('phpunit', 'ListUploadsTest');
+        // need to write more than 4k to trigger a satellite call
+        $upload->write(random_bytes(5_000));
```

Comment:
it can be also simulated with BeginUpload call

Reply:
Added a comment. This isn't in the binding yet.